### ISSUE TITLE
Craft destroyed/spinning when timewarping prelaunch

### ISSUE THREAD

**Issue by UnheardWolf344** [mockingbirdnest/Principia#4118]
I found a weird bug. Whenever I on-rails timewarp on the launchpad prelaunch (three crafts have all displayed this behavior, all using modularlaunchpads static test stands), my craft begins spinning uncontrollably along one axis, clipping through the launchpad as though physics were turned off (second & third pic). The camera cannot move. If I try to quicksave, it does nothing. holding f9  does nothing. when I press esc multiple times, the camera zooms as seen in the fourth picture. f5 and f9 still do nothing. If I press f3, then close that menu, f5 says "vessel destroyed. Cannot quicksave at this point." and f9 allows me to actually quickload. Any ideas? I assume that it's something to do with the static launch stands, as this doesn't happen on crafts using the circular launch stand or the milkstool launch stand. 
For the first occurrence (pics one, two, and four) i have a ksp.log, with the error starting at `EXC 22:24:17.946`. I have confirmed the error to not be resolved by removing deepfreeze.
For the second occurrence (thirdpic) I have a ksp.log, stderr.log, and INFO.log (the one that displays the error clearly in the second half of the log, referencing #1421 ).

[INFO.20241016-185847.11876.log](https://github.com/user-attachments/files/17404732/INFO.20241016-185847.11876.log)
[stderr.log](https://github.com/user-attachments/files/17404740/stderr.log)

[KSP (first craft).log](https://github.com/user-attachments/files/17404724/KSP.first.craft.log)
[KSP ( second craft).log](https://github.com/user-attachments/files/17404738/KSP.second.craft.log)

![screenshot116](https://github.com/user-attachments/assets/ce56c759-172b-4fbb-8c42-98030bcd0614) the "no target" message that pops up upon first timewarping. The craft does not spin during timewarp, but rather only when timewarp is not active. If timewarp starts while it is spinning, it just stays in the partially rotated position that it was in when the timewarp began.
![screenshot117](https://github.com/user-attachments/assets/73102359-2298-4ab9-b5b6-75e35224e813)
spinning 
![screenshot124](https://github.com/user-attachments/assets/158dc249-6e09-40eb-8f8a-582bce499765)
spinning with another craft

![screenshot119](https://github.com/user-attachments/assets/806a1f3e-0961-4ab3-b2b0-1654097bac1b)
zoomed spinning

**Comment by pleroy** [comment]
I don't see that this has anything to do with Principia.  Surely Principia is not going to cause F5 or F9 to stop working.  Also Principia does nothing with vessels that are grounded.  You tell us that the problem only occurs with the Modular Launch Pads static test stands, so maybe open an issue with Modular Launch Pads.

PS: Don't try to interpret the Principia logs.  The messages about the vessel not being ready is expected when in prelaunch.

**Comment by UnheardWolf344** [comment]
First, principia does not prevent f5 or f9 to stop working; it merely seems to "destroy" the vessel, which then makes quicksaving not work, as you cannot quicksave when piloting no vessel. I believe that this "destroy"ing of the vessel happens due to the logged event "Skipping AdvanceTime and SetBodyFrames while waiting for the vessel to be fully ready (see #1421)." Principia, at least from my understanding, seems to not think the vessel is ready, so an issue is caused.
Second, my interpretation of the principia logs is due to the fact that 1. The message about the vessel not being ready begins at almost exactly the same time as the Null Reference Exception spam in the ksp.log, and 2. it continues all the way until I closed the game, which I did while the craft was still spinning. As such, I made the assumption that the logged message was directly related to the issue.
